### PR TITLE
Add output-doctype exist serialization option

### PR DESCRIPTION
**Pull request by adamretter**
* Add output-doctype exist serialization option
* options are not boolean but actually "yes" | "no" values

### CODE REVIEW

File: src/main/xar-resources/data/xquery/xquery.xml [hunk 1]
```diff
@@ -607,7 +607,13 @@
                     </listitem>
                 </varlistentry>
                 <varlistentry>
-                    <term> <literal>expand-xincludes : ( boolean )</literal> </term>
+                    <term> <literal>output-doctype : ( "yes" | "no" )</literal> </term>
+                    <listitem>
+                        <para>Must the serializer expand XInclude elements?</para>
```
Comment (joewiz) [inline src/main/xar-resources/data/xquery/xquery.xml:612]: This description appears to be unchanged from the `expand-xincludes` explanation.

Comment (adamretter) [reply src/main/xar-resources/data/xquery/xquery.xml]: Thanks Joe - not sure what happened. I have fixed it now.

Comment (joewiz) [reply src/main/xar-resources/data/xquery/xquery.xml]: Thanks, Adam!

**Review by joewiz — APPROVED** [review]

**Comment by joewiz** [comment]
This can be merged as soon as https://github.com/eXist-db/exist/pull/4547 is.

**Comment by joewiz** [comment]
https://github.com/eXist-db/exist/pull/4547 has been merged. Thank you @adamretter!

**Pull request merged**